requirements-py3.txt

```diff
@@ -1 +1,2 @@
 python-dateutil
+python3-PyYAML
```

Comment:
"python3-PyYAML" isn't a PyPI module name. Just PyYAML is fine, that works on both Python 2 and Python 3.

Reply:
ok deleting